bamboo/integration_tests/test_integration_lenet.py

```diff
@@ -14,6 +14,15 @@
 sys.path.insert(0, os.path.join(os.path.dirname(current_dir), 'common_python'))
 import tools
 
+# Applications directories
+bamboo_dir = os.path.dirname(current_dir)
+lbann_dir = os.path.dirname(bamboo_dir)
+sys.path.insert(0, os.path.join(lbann_dir, 'applications', 'vision'))
+import data.mnist
```

Comment:
I'd suggest copy-pasting the MNIST and ImageNet scripts into `bamboo/common_python`. The Bamboo framework is going to be horribly messy and complicated (until we have build-based testing) and it would be a bad idea for that messiness to contaminate the application directory. After all, the LeNet model is supposed to be simple and useful for learning how to use LBANN.

Reply:
Done.